koku/koku/test_trino_db_utils.py

```diff
@@ -146,6 +146,46 @@ def cursor(self):
             conn = FakerFakeTrinoConn()
             executescript(conn, sqlscript)
 
+    def test_retry_logic_on_no_such_key_error(self):
+        class FakerFakeTrinoCur(FakeTrinoCur):
+            execute_calls = 0
+
+            def execute(self, *args, **kwargs):
+                self.__class__.execute_calls += 1
+                if self.__class__.execute_calls == 1:
```

Comment:
This should be an instance attribute, not a class attribute.
```suggestion
        class FakerFakeTrinoCur(FakeTrinoCur):
            def __init__(self, *args, **kwargs):
                self.execute_calls = 0

            def execute(self, *args, **kwargs):
                self.execute_calls += 1
                if self.execute_calls == 1:
```